torch/_torch_docs.py

```diff
@@ -5719,3 +5719,30 @@ def parse_kwargs(desc):
     Tensor: A 1-D tensor of size :math:`(\text{{window_length}},)` containing the window
 
 """.format(**factory_common_args))
+
+
+add_docstr(torch.meshgrid,
+           r"""
+meshgrid(seq) -> seq
+
+Create N N-dimensional grid coordinates with the ith coordinates defined by the ith vector.
```

Comment:
Please expand a little more on what meshgrid does (I'm having a hard time parsing this). It would be good if you define `N` (is N the number of inputs?)